app/Http/Controllers/API/FamiliaProfesionalController.php

```diff
@@ -37,6 +39,7 @@ public function store(Request $request)
      */
     public function show(FamiliaProfesional $familiaProfesional)
     {
+        $familiaProfesional->load('ciclos');
```

Comment:
Para mostrar los ciclos asociados a las familias profesionales has utilizado `->load`, en cambio, para mostrar la familia profesional aasociada a un ciclo has utilizado la _propiedad dinámica_ `$this->familiaProfesional` en `CicloResource`.

Además, esta solución NO la has aplicado en el index, por lo que los registros del `GET FamiliasProfesionales` aparecerían sin los ciclos asociados.